### PR TITLE
Add documentation for HyperLogLog functions

### LINKED ISSUE #8968: Document HLL-related functions
We need docs for `approx_set`, `empty_approx_set`, `merge` and `cardinality`.

### PR DESCRIPTION
**Pull request by jagill**
This includes a new function page hyperloglog, as well as modifications
to other pages.  Fixes #8968

### CODE REVIEW

**Comment by arhimondr** [comment]
@jagill Thanks for doing this ;-)

File: presto-docs/src/main/sphinx/functions/hyperloglog.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Serialization
+-------------
+
+Data sketches can be serialized to and deserialized from ``varbinary``. This allows them to be stored for later use.
+Combined with the ability to merge multiple sketches, this allows one to calculate :func:`approx_distinct`
+of the elements of a partition of a query, then for the entirety of a query with very little cost. For example,
+calculating the ``HyperLogLog`` for daily unique users will allow weekly or monthly unique
+users to be calculated incrementally by comining the dailies. This is similar to computing weekly revenue by summing
+daily revenue. Uses of :func:`approx_distinct` with ``GROUPING SETS``, can be converted to use
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/functions/hyperloglog.rst]: Let's remove the comma after `GROUPING SETS`

**Comment by tdcmeehan** [comment]
Other than the nits above I think it looks good.  @electrum did you have any further feedback?

**Comment by jagill** [comment]
I realize I didn't write anything about `empty_approx_set` (because I only just now read that it exists).  So I suppose that documentation will still be missing.

**Comment by electrum** [comment]
Merged, thanks!

**Comment by electrum** [comment]
I made some minor edits including adding `empty_approx_set` and had @tdcmeehan do a review of the rendered docs.

**Pull request closed without merging**